src/packer.pkr.hcl

```
@@ -46,89 +46,98 @@ variable "skip_create_ami" {
   type        = bool
 }
 
-data "amazon-ami" "debian_bullseye" {
+data "amazon-ami" "windows" {
   filters = {
-    name                = "debian-11-amd64-*"
+    name                = "Windows_Server-2022-English-Full-Base-*"
     root-device-type    = "ebs"
     virtualization-type = "hvm"
   }
   most_recent = true
-  owners      = ["136693071363"]
+  owners      = ["amazon"]
   region      = var.build_region
 }
 
 locals { timestamp = regex_replace(timestamp(), "[- TZ:]", "") }
 
-source "amazon-ebs" "example" {
+source "amazon-ebs" "windows" {
   ami_block_device_mappings {
     delete_on_termination = true
     device_name           = "/dev/xvda"
     encrypted             = true
     volume_size           = 8
     volume_type           = "gp3"
   }
-  ami_name                    = "example-hvm-${local.timestamp}-x86_64-ebs"
+
+  ami_name                    = "windows-commando-hvm-${local.timestamp}-x86_64-ebs"
```

I don't think we need the "hvm" bit in our AMI name here:
```suggestion
  ami_name                    = "windows-commando-${local.timestamp}-x86_64-ebs"
```

We might not even want to mention "commando" yet until you get to the point of installing it, just to eliminate any confusion with the AMIs that have been produced.

---

Changed in https://github.com/cisagov/windows-commando-packer/pull/10/commits/dc820cfab375d9150fdd8042188bb5be993518e2.